x-pack/plugins/fleet/cypress/e2e/uninstall_token.cy.ts

```diff
@@ -74,6 +74,49 @@ describe('Uninstall token page', () => {
     cy.getBySel(UNINSTALL_TOKENS.POLICY_ID_TABLE_FIELD).should('have.length', 1);
   });
 
+  it('should show token when policy is removed', () => {
```

Comment:
as these are exactly the same test cases as the ones above, what do you think about putting them into a `describe.each()` passing `()=>{}` or `() => cleanupAgentPolicies()` as a parameter, in order to ease maintenance?

Reply:
done! https://github.com/elastic/kibana/pull/166701/commits/c226e71a791533703814abbc5b49d59fb14d0629